src/logging.rs

```diff
@@ -60,10 +73,58 @@ pub fn setup_layers(
         .with_writer(std::io::stderr)
         .with_filter(EnvFilter::from_env("AZURE_INIT_LOG"));
 
+    let log_path = config
+        .map(|cfg| cfg.telemetry_log_path.path.clone())
+        .unwrap_or_else(|| PathBuf::from(DEFAULT_TELEMETRY_LOG_PATH));
+
+    let file_layer = match OpenOptions::new()
+        .create(true)
+        .append(true)
+        .open(&log_path)
```

Comment:
One thing that we didn't discuss is the permission for azure-init datadir and azure-init logpath? Using cloud-init as a reference, I think different distros have taken different opinions on this, with Ubuntu's cloud-init.log being visible to all users while Suse has limited cloud-init.log access to only root user?

@cjp256 @jeremycline to comment

Reply:
would be best to ensure chmod 600 for that reason.

cloud-init.log is chmod 640, with group owner "adm" which gives a little more balance as any user with "adm" privileges can see it too.